### PR TITLE
RMF-419 Nov 23 release notes Update Introduction

### PR DESCRIPTION
**Pull request by andreamussap**
API Gateway - http://pr-2753.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20231130_apimgr_relnotes/

API Portal - http://pr-2753.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20231130_apip_relnotes/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | 32063ce1 |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-2753.axway-open-docs.opendocs-builder.pcloud.axway.int |

**Review by jmbermingham — APPROVED** [review]
Looks good now thanks.

File: content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ description: null
...
 API Gateway and API Manager updates are cumulative, comprising new features and changes delivered in previous updates unless specifically indicated otherwise in the Release notes.
 
-placeholder
+The key feature delivery for this update is the support of GraphQL, which has been the highest voted item on our [Ideas portal](https://amplifyideas.axway.com/?project=APIG&sort=popular) for some time. This support enables customers to import GraphQL specifications and apply security and policies, as well as view the specification in API Manager and try it out.
+
+In this update we have also done further work on the Helm charts for container deployment, making them more flexible and enabling new features.
+
+Support for newer base operating systems (OS) has also been delivered - RHEL 9, Oracle Linux 9, and SUSE 15 are now supported. For our Axway images used for container deployment, these have been upgraded to use Red Hat Universal Base Images (UBI) 9 as the base OS.
+
+Finally, we have completed some items as a follow up to the Java upgrade in the August 23 Update, for example, replacement of the Apache commons email with Java simple mail.
```
[the remark this answers is not in the document]
Comment (andreamussap) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:16]: @jmbermingham do I delete this sentence then?

[the remark this answers is not in the document]
Comment (jmbermingham) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:16]: Yes, delete this sentence - I did not realise we were not going to get it in. If we do, we can add it back in at the end.

**Pull request merged**